app/src/services/version.js

```diff
@@ -90,7 +90,8 @@ const service = {
           createdBy: userId,
           deleteMarker: data.deleteMarker,
```

Comment:
I think you need to add lastModifiedDate to the db insert happening on line 48 of services/version.js when it does the copy process. This is invoked during the flow when updating/replacing metadata when the bucket doesnt have versioning.

Reply:
separate note.. i hope the lastModifiedDate for a version doesnt change when you edit tags on that version 😬

Reply:
Good catch. Looking into adding and updating the copy service to account for the new value. Will investigate tag modification signatures.